kratos/python/add_geometries_to_python.cpp

```diff
@@ -145,6 +153,9 @@ void  AddGeometriesToPython(pybind11::module& m)
     .def("Area",&GeometryType::Area)
     .def("Volume",&GeometryType::Volume)
     .def("__str__", PrintObject<GeometryType>)
+    .def("__getitem__", [](GeometryType& self, unsigned int i){return self(i);} )
+    .def("__iter__",    [](GeometryType& self){return py::make_iterator(self.begin(), self.end());},  py::keep_alive<0,1>())
+    .def("__len__",     [](GeometryType& self){return self.PointsNumber();} )
```

Comment:
Great job here. This is exactly what I was missing, as stated in #6433. However, I do not see any test were `GetGeometry()` is called for an element and then it is iterated, or some node is accessed through `[]`. That will probably be one of the most common uses of this iterable Geometry once this PR is approved, so it would be great to have at least one call to it.